game/theater/base.py

```diff
@@ -37,6 +39,10 @@ def total_aircraft(self) -> int:
     def total_armor(self) -> int:
         return sum(self.armor.values())
 
+    @property
+    def total_ground_units(self) -> int:
+        return sum(self.armor.values()) + sum(self.aircraft.values())
```

Comment:
I suppose planes are ground units while their parked, but I don't think this is what you wanted :)